docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb

```diff
@@ -20,7 +20,7 @@
     "1. Open your workspace\n",
```

Comment:
<p>Recommend:</p><ul><li>which may -&gt; that may</li><li>indentify -&gt; identify</li><li>Changing instances of "popup" to "pop up"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>